macros/CompatBeta.ejs

```diff
@@ -496,7 +533,7 @@ function writeCompatCells(supportData) {
       output += ' bc-has-history';
     }
 
-    output += `">${supportInfo}`;
+    output += `"><span class="bc-browser-name">${browserName}</span>${supportInfo}`;
```

Comment:
Having this hidden for now works for me. Note that we want to remove icons and use text everywhere and not just on mobile in the future. See https://github.com/mdn/kumascript/pull/616. I'm not sure when we want to do this, though. Maybe now isn't the time for it. So this is OK with me :+1:

Reply:
Agreed. The scope of this was mobile specific, as that is where we will mostly see the use case of people turning of custom fonts.